units/host_termination.go

```diff
@@ -400,10 +400,12 @@ func (j *hostTerminationJob) checkAndTerminateCloudHost(ctx context.Context) err
 	}
 
 	if cloudInfo.Status == cloud.StatusTerminated {
-		catcher := grip.NewBasicCatcher()
-		catcher.Errorf("host is already terminated in the cloud: '%s'", cloudInfo.StateReason)
-		catcher.Wrap(j.host.Terminate(ctx, evergreen.User, "cloud provider indicated that host was already terminated"), "marking host as terminated")
-		return catcher.Resolve()
+		grip.Info(message.Fields{
```

Comment:
Is this something we want to investigate further or action on? If yes, can we open a ticket for that? 
If we don't plan to take any further action, I wonder if we should maybe remove this altogether consider the volume of logs we get for it.

Reply:
I looked at it a little bit and I don't think it indicates any deeper issue for now, just that it logs more often due to higher global max hosts and more aggressive host allocation (which results in more host termination). If no other errors pop up, I don't think there's anything to investigate and we should just continue monitoring the scheduler/allocator changes.

I do think it's useful to trace when a host that's already terminated is being terminated again. It doesn't hurt to keep around since it's not a significant source of logs and we can always remove later on if we think it's problematic.